tests/invalid/CoCoInternalAssignedOutsideBlock.nestml

```diff
@@ -0,0 +1,40 @@
+"""
+CoCoInternalAssignedOutsideBlock.nestml
```

Comment:
I think we should do this for the inline expression coco as well. I just found that `CoCoAssignmentToInlineExpression.nestml` is not being called in the `coco_test.py`. According to the new coco, this file should fail.